crates/ra_syntax/src/parsing/reparsing.rs

```diff
@@ -41,37 +41,42 @@ fn reparse_token<'node>(
     root: &'node SyntaxNode,
     edit: &AtomTextEdit,
 ) -> Option<(GreenNode, TextRange)> {
-    let token = algo::find_covering_element(root, edit.delete).as_token()?.clone();
-    match token.kind() {
+    let prev_token = algo::find_covering_element(root, edit.delete).as_token()?.clone();
```

Comment:
Spent some time figuring out what is going on in this method to properly integrate changes from lexer. The core implementation here didn't change, just renamed some variables for clarity...

Reply:
Good names do a lot for code readability :+1: